frontend/src/routes/wallet/keypair/taint.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState, useCallback } from 'react'
+import { Intent } from '../../../config/intent'
+import { Header } from '../../../components/header'
+import { Button } from '../../../components/button'
+import { AppToaster } from '../../../components/toaster'
+import { BreakText } from '../../../components/break-text'
+import { KeyValueTable } from '../../../components/key-value-table'
+import { useCurrentKeypair } from '../../../hooks/use-current-keypair'
+import { requestPassphrase } from '../../../components/passphrase-modal'
+import { updateKeyPairAction } from '../../../contexts/global/global-actions'
+import { useGlobal, GlobalDispatch } from '../../../contexts/global/global-context'
+import { createLogger } from '../../../lib/logging'
+import { Service } from '../../../service'
+
+const logger = createLogger('Taint')
+
+const useTaint = (dispatch: GlobalDispatch, pubKey?: string, wallet?: string) => {
+  const [loading, setLoading] = useState(false);
```

Comment:
question (non-blocking): Do we even need this loading state? I'd expect the promise to resolve almost instantly

Reply:
i don't know the reality of how fast this should resolve, i think it definitely can't hurt anyone to disable the taint / untaint on any async operation so the user's can't click the button like crazy, just in case.

Reply:
Yeah your right. Lets leave it in